.github/workflows/trigger-all.yml

```diff
@@ -0,0 +1,44 @@
+name: build and test jobs for nightly and release
+run-name: ${{ github.actor }} triggered nightly or release on ${{ github.ref }}
+on:
+    schedule:
+      # * is a special character in YAML so you have to quote this string
+      - cron: '30 0 * * *'  # nightly run
+
+    workflow_dispatch:
+        inputs:
+            wf_category:
+                description: "workflow category, default is NIGHTLY"
+                type: choice
+                options:
+                    - NIGHTLY
+                    - RELEASE
+                default: NIGHTLY
+            push_to_pypi:
+                description: "when set and tests pass, then '.whl' & '.tar.gz' will be pushed to public pypi"
+                type: boolean
+                default: true
+            gitref:
+                description: "git commit hash or tag name" 
+                type: string
+                default: 'main'
+
+    push:
+        branches:
+          - auto
+
+jobs:
+
+    BUILD-TEST:
+        uses: ./.github/workflows/build-test.yml
+        name: ${{ inputs.wf_category || 'RELEASE' }}
```

Comment:
This (and line 36) mean the default will be `RELEASE` (e.g., when the job is triggered by the nightly cron schedule, it will be a `RELEASE` job), is that the desired state? I would think that `NIGHTLY` should be the default for the cron-scheduled run.

Reply:
Yes good catch, actually it'll be NIGHTLY as its default value. I was testing the RELEASE workflow and haven't changed it back yet. I'll update it back to NIGHTLY after the RELEASE run is finished. Thanks.

Reply:
updated.